### ISSUE TITLE
Kodi 19-beta search breaks loading history

### ISSUE THREAD

**Issue by anrxc** [anxdpanic/plugin.video.youtube#85]
### Context

Kodi 19-beta
YouTube Plugin 6.8.9+matrix.1

My search history contains what I am guessing are either cp1250 or 8859-2 characters (eg. ćšđćž). Popup error would display. Search history would be empty. New searches were not possible.

### Temporary workaround

```
--- resources\lib\youtube_plugin\kodion\utils\storage.py
+++ resources\lib\youtube_plugin\kodion\utils\storage.py

-return _decode(item[1]), item[0]

+try:
+    return _decode(item[1]), item[0]
+except:
+    return
```

### Original error

```
File "C:\Users\User\AppData\Roaming\Kodi\addons\plugin.video.youtube\resources\lib\youtube_plugin\kodion\utils\storage.py", line 215, in _get
  return _decode(item[1]), item[0]
File "C:\Users\User\AppData\Roaming\Kodi\addons\plugin.video.youtube\resources\lib\youtube_plugin\kodion\utils\storage.py", line 200, in _decode
  return pickle.loads(obj)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc5 in position 0: ordinal not in range(128)
-->End of Python script error report<--
```

**Comment by vtolstov** [comment]
any news?

**Comment by anrxc** [comment]
I applied my ugly fix to 6.8.10+matrix.1~alpha1 in Kodi 19 stable as well.
